java-strings-ops-2/src/main/java/com/baeldung/string/Anagram.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.string;
+
+import java.util.Arrays;
+import com.google.common.collect.HashMultiset;
+import com.google.common.collect.Multiset;
+
+public class Anagram {
+    public boolean isAnagramSort(String str1, String str2) {
+        if (str1.length() != str2.length()) {
+            return false;
+        }
+        char[] a1 = str1.toCharArray();
+        char[] a2 = str2.toCharArray();
+        Arrays.sort(a1);
+        Arrays.sort(a2);
+        return Arrays.equals(a1, a2);
+    }
+
+    public boolean isAnagramCouting(String str1, String str2) {
+        if (str1.length() != str2.length()) {
+            return false;
+        }
+        final int MAX_CHARACTER = 256;
```

Comment:
Let's not declare the `final` here - this should be a `private static final` constant for the whole class.

It's also worth pointing out somewhere, that this algorithm will fail with `UTF-8` data. I'm reviewing the code before the article... but perhaps a comment in the code will also help an unwary reader.